packages/Php74/src/Rector/Property/TypedPropertyRector.php

```diff
@@ -126,7 +129,44 @@ private function removeVarPhpTagValueNodeIfNotComment(Property $property): void
             return;
         }
 
+        // keep string[] etc.
+        if ($this->isNonBasicArrayType($property, $varTagValueNode)) {
```

Comment:
What if it is non-array generic node?

```
/**
 * @var Traversable<int, string>
 */
private Traversable $foo;
```

or even non-iterable one?

```
/**
 * @var Promise<Foo>
 */
private Promise $foo;
```

Maybe the code already covers it but the method names shouldn't contain the word "array" imo.

Reply:
I'll look on it, thanks

Reply:
Added :+1: